akka-persistence-typed/src/test/java/jdocs/akka/persistence/typed/BasicPersistentBehaviorTest.java

```diff
@@ -4,29 +4,22 @@
 
 package jdocs.akka.persistence.typed;
 
+import akka.actor.typed.ActorRef;
 import akka.actor.typed.Behavior;
 import akka.actor.typed.SupervisorStrategy;
 import akka.actor.typed.javadsl.ActorContext;
 import akka.actor.typed.javadsl.Behaviors;
-import akka.persistence.typed.DeleteEventsFailed;
-import akka.persistence.typed.DeleteSnapshotsFailed;
-import akka.persistence.typed.RecoveryCompleted;
-import akka.persistence.typed.SnapshotFailed;
-import akka.persistence.typed.javadsl.CommandHandler;
-import akka.persistence.typed.javadsl.EventHandler;
-// #behavior
-import akka.persistence.typed.javadsl.EventSourcedBehavior;
-import akka.persistence.typed.PersistenceId;
-
-// #behavior
-import akka.persistence.typed.javadsl.RetentionCriteria;
-import akka.persistence.typed.javadsl.SignalHandler;
+import akka.persistence.typed.*;
+import akka.persistence.typed.javadsl.*;
 
 import java.time.Duration;
 import java.util.ArrayList;
 import java.util.List;
 import java.util.Set;
 
+// #behavior
+// #behavior
```

Comment:
Looks like something (I think IntelliJ Organize Imports) removed the imports from the sample?

Reply:
oh evil, I'll check thanks!

Reply:
Fixed 👍